app/src/Routes/Works/Works.js

```diff
@@ -1,9 +1,30 @@
 import React from 'react';
 import Typography from '@material-ui/core/Typography';
+import * as works from './works.json';
+
+const Piece = (piece) => (
+    <li key={piece.title + piece.type}>
+        <div>{piece.title}</div>
+        <div>{piece.type}</div>
+        {piece.instruments ? piece.instruments.join(",") : "" }
```

Comment:
oh one more thing - i think i'd do something like this instead here:
`{piece.instruments && piece.instruments.join(',')}`